Mathlib/Data/String/Defs.lean

```diff
@@ -82,7 +82,7 @@ def popn (s : String) (n : Nat) : String :=
   ⟨s.toList.drop n⟩
 #align string.popn String.popn
 
-/-- Produce the head character from the string `s`, if `s` is not empty, otherwise 'A'. -/
+/-- Produce the head character from the string `s`, if `s` is not empty, otherwise `'A'`. -/
```

Comment:
I don't think this one is helpful.

Reply:
@semorrison But this is consistent with the following code: https://github.com/leanprover/lean4/blob/f9da1d8b55ca6989297fb952985b7d8d6c77e14b/src/Init/Prelude.lean#L3709